release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -88,6 +88,21 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-networking"]
 === Networking
 
+[id="ocp-4-10-networking-ovn-gateway-config"]
+==== OVN-Kubernetes support for gateway configuration
+
+The OVN-Kubernetes CNI network provider adds support for configuring how egress traffic is sent to the node gateway.
+By default, egress traffic is processed in OVN to exit the cluster and traffic is not affected by specialized routes in the kernel routing table.
+
+This enhancement adds a `gatewayConfig.routingViaHost` field.
+The field can be set at runtime as a post-installation activity and when it is set to `true`, egress traffic is sent from pods to the host networking stack.
+This enhancement benefits highly-specialized installations and applications that rely on manually configured routes in the kernel routing table.
+
+This enhancement has an interaction with the Open vSwitch hardware offloading feature.
```

Comment:
Saw Samantha's comment about the case of Open vSwitch in earlier PR.

Reply:
Open vSwitch is used throughout the openshift-docs repo, so I think the capitalization is correct.